env/fs_posix.cc

```diff
@@ -833,7 +839,15 @@ class PosixFileSystem : public FileSystem {
     optimized.fallocate_with_keep_size = true;
     return optimized;
   }
-
+#ifdef OS_LINUX
+  Status RegisterDbPaths(const std::vector<std::string>& paths) override {
+    return logical_block_size_cache_.RefAndCacheLogicalBlockSize(paths);
+  }
+  Status UnregisterDbPaths(const std::vector<std::string>& paths) override {
+    logical_block_size_cache_.UnrefAndTryRemoveCachedLogicalBlockSize(paths);
+    return Status::OK();
```

Comment:
Can we avoid the return code? In that way we don't have to handle the hypothesis case that it fails in ColumnFamilyData's destructor.

Reply:
Since `UnregisterDbPaths` is a general API, in case other implementations might need to return a status, I prefer to keep the status here.